Updating discord_voice_channel causes plugin crash

For a resource with the following definition:
```hcl
resource discord_voice_channel staff_voice {
  name                     = "staff-chat"
  server_id                = discord_server.nlp.id
  category                 = discord_category_channel.staff.id
  sync_perms_with_category = true
}
```
attempting to update the resource (for example, changing the channel name) crashes the plugin. Logs:
```
Terraform will perform the following actions:

  # discord_voice_channel.staff_voice will be updated in-place
  ~ resource "discord_voice_channel" "staff_voice" {
        id                       = "1035987669412487258"
      ~ name                     = "staff-chat" -> "staff-chat-test"
        # (7 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

[...]

│ Error: Request cancelled
│ 
│   with discord_voice_channel.staff_voice,
│   on main.tf line 141, in resource "discord_voice_channel" "staff_voice":
│  141: resource discord_voice_channel staff_voice {
│ 
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.

Stack trace from the terraform-provider-discord_v1.1.0 plugin:

panic: interface conversion: interface {} is int, not uint

goroutine 64 [running]:
github.com/lucky3028/discord-terraform/discord.resourceChannelUpdate({0x1b3a668, 0xc0006262a0}, 0xc000290300, {0x1863620?, 0xc0004e6a10?})
        github.com/lucky3028/discord-terraform/discord/resource_discord_channel.go:287 +0x1608
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).update(0xc00024ec40, {0x1b3a6a0, 0xc0003aac60}, 0xd?, {0x1863620, 0xc0004e6a10})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:741 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc00024ec40, {0x1b3a6a0, 0xc0003aac60}, 0xc00047bba0, 0xc000290180, {0x1863620, 0xc0004e6a10})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:847 +0x83a
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000457f08, {0x1b3a6a0?, 0xc0003aab40?}, 0xc000754550)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/grpc_provider.go:1021 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc000273680, {0x1b3a6a0?, 0xc0003aa270?}, 0xc000200cb0)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:818 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x19b2b80?, 0xc000273680}, {0x1b3a6a0, 0xc0003aa270}, 0xc000200c40, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001e21e0, {0x1b3e7c0, 0xc00029a9c0}, 0xc00035cc60, 0xc000470a20, 0x20add60, 0x0)
        google.golang.org/grpc@v1.48.0/server.go:1295 +0xb2b
google.golang.org/grpc.(*Server).handleStream(0xc0001e21e0, {0x1b3e7c0, 0xc00029a9c0}, 0xc00035cc60, 0x0)
        google.golang.org/grpc@v1.48.0/server.go:1636 +0xa2f
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.48.0/server.go:932 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.48.0/server.go:930 +0x28a

Error: The terraform-provider-discord_v1.1.0 plugin crashed!
```

--- a/discord/resource_discord_channel.go
+++ b/discord/resource_discord_channel.go
@@ -284,8 +284,8 @@ func resourceChannelUpdate(ctx context.Context, d *schema.ResourceData, m interf
 		}
 	case "voice":
 		{
-			bitRate = map[bool]uint{true: d.Get("bitrate").(uint), false: channel.Bitrate}[d.HasChange("bitrate")]
-			userLimit = map[bool]uint{true: d.Get("user_limit").(uint), false: channel.UserLimit}[d.HasChange("user_limit")]
+			bitRate = map[bool]uint{true: uint(d.Get("bitrate").(int)), false: channel.Bitrate}[d.HasChange("bitrate")]
+			userLimit = map[bool]uint{true: uint(d.Get("user_limit").(int)), false: channel.UserLimit}[d.HasChange("user_limit")]
 		}
 	}
 

